### PR TITLE
Add basic RW style colours to the Assets catalog

### PR DESCRIPTION
**Pull request by liamnichols**
This change introduced RW styled colours to the asset catalog to help make things easier when creating a nice looking sample project. I also grouped assets (launch-assets, colors) to make the catalog a little cleaner. 

screenshot 2018-11-04 at 14 19 48

I recently used this approach in a tech edit and @jrturton suggested that we add it to the default project so you can decide here if it's worth it.. 

I just took the three colours from the website but if there is a better design spec to follow then I can update things. Colours included: 

- **rw-green:** `#006837`, the green colour from the logo
- **rw-dark:** `#333333`, website background colour in dark mode or the standard text colour
- **rw-light:** `#F2F6FA`, website background colour in light mode

### CODE REVIEW

**Comment by rcritz** [comment]
@liamnichols please change the RW-green to #006837. That’s the actual color from the designer. 

Thanks for doing this.

File: MyRWTutorial/Assets.xcassets/colors/rw-green.colorset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "version" : 1,
+    "author" : "xcode"
+  },
+  "colors" : [
+    {
+      "idiom" : "universal",
+      "color" : {
+        "color-space" : "srgb",
+        "components" : {
+          "red" : "0x15",
```
Comment (liamnichols) [inline MyRWTutorial/Assets.xcassets/colors/rw-green.colorset/Contents.json]:
```suggestion
          "red" : "0x00",
```

File: MyRWTutorial/Assets.xcassets/colors/rw-green.colorset/Contents.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+          "alpha" : "1.000",
+          "blue" : "0x43",
+          "green" : "0x84"
```
Comment (liamnichols) [inline MyRWTutorial/Assets.xcassets/colors/rw-green.colorset/Contents.json]:
```suggestion
          "green" : "0x68"
```

File: MyRWTutorial/Assets.xcassets/colors/rw-green.colorset/Contents.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  },
+  "colors" : [
+    {
+      "idiom" : "universal",
+      "color" : {
+        "color-space" : "srgb",
+        "components" : {
+          "red" : "0x15",
+          "alpha" : "1.000",
+          "blue" : "0x43",
```
Comment (liamnichols) [inline MyRWTutorial/Assets.xcassets/colors/rw-green.colorset/Contents.json]:
```suggestion
          "blue" : "0x37",
```

**Review by liamnichols — COMMENTED** [review]
Feedback on rw-green colour

**Comment by liamnichols** [comment]
Done @rcritz, please ♻️

**Pull request merged**